blocks/columns/columns.css

```diff
@@ -6,6 +6,30 @@
     justify-content: center;
 }
 
+.columns > div > div p:nth-child(2) {
```

Comment:
I'm always worried when I see CSS that depends on content in a certain order...
Like with cards, I'd try to think semantics rather than number.

Reply:
switched to semantics